docs/connectors/kafka-connect-connector.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Kafka Connect Connectors with Conduit"
+sidebar_position: 8
+---
+
+# Using Kafka Connect Connectors with Conduit
+
+The [Conduit Kafka Connect Wrapper connector](https://github.com/ConduitIO/conduit-kafka-connect-wrapper) is a special
+connector that allows you to use [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.html) 
+connectors with Conduit. Conduit doesn't come bundled with Kafka Connect connectors, but you can use it to bring any
+Kafka Connect connector with Conduit.
+
+This connector gives you the ability to:
+
+- Easily migrate from Kafka Connect to Conduit.
+- Remove Kafka as a dependency to move data between data infrastructure.
+- Leverage a datastore if Conduit doesn't have a native connector.
+
+Since the Conduit Kafka Connect Wrapper itself is written in Java, but most of Conduit's connectors are written in Go,
+it also serves as a good example of the flexibility of the Conduit Plugin SDK.
+
+Let's begin.
+
+## How it works
+
+To use the Kafka Connect wrapper connector, you'll need to:
+
+1. Clone the [`conduit-kafka-connect-wrapper`](https://github.com/ConduitIO/conduit-kafka-connect-wrapper) repository.
+1. Build the Connector JAR.
+1. Download Kafka Connect JARs and any dependencies you would like to add.
+1. Create a pipeline configuration file.
+1. Run Conduit.
+
+
+## Setup
+
+To begin, clone the connector:
+
+```
+git clone git@github.com:ConduitIO/conduit-kafka-connect-wrapper.git
+```
+
+Then, we need to build the connector. The Kafka Connect wrapper connector is written in Java, so it needs to be compiled.
+
+
+```bash
+cd conduit-kafka-connect-wrapper
+
+./scripts/dist.sh
+```
+
+This script will:
+
+1. Create a directory called `dist`.
+1. Compile the connector and place the JAR in `dist`.
+1. Create `dist/libs` directory.
+
+Now that we have everything setup, we can add the Kafka Connect connectors.
+
+## Adding Kafka Connect Connector
+
+The `libs` directory is where you put the Kafka Connect connector JARs and their dependencies (if any). The wrapper 
+plugin will automatically load connectors and all the other dependencies from a `libs` directory.
+
+For this example, we will use the PostgreSQL Kafka Connect JDBC Connector. To install, add the following:
```

Comment:
We can also mention that we have [this script](https://github.com/ConduitIO/conduit-kafka-connect-wrapper/blob/main/scripts/download-connector.sh), that makes it a bit easier to download a connector and its dependencies.